src/main/kotlin/frc/robot/lib/MarkerCommands.kt

```diff
@@ -0,0 +1,18 @@
+package frc.robot.lib
+
+import edu.wpi.first.wpilibj.Alert
+import edu.wpi.first.wpilibj.DataLogManager
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+
+const val ABNORMAL_EVENT_NAME = "PROBLEM!"
+val alert = Alert(ABNORMAL_EVENT_NAME, Alert.AlertType.kWarning)
+
+fun markEvent(eventName: String): Command = Commands.runOnce({
```

Comment:
Can top-level functions in Kotlin be private? If yes, then this one should be private.

Reply:
920e64150027ab79e600bc7fcca4bd1bc8aef948